### DISCUSSION TITLE
1.1.0 Release

### DISCUSSION THREAD

**Discussion by janjaali** [apache/pekko-persistence-jdbc#222] [General]
Hi, are there any plans for an upcoming 1.1.0 release? We have noticed that you are currently working on a Pekko 1.1.1 and there is already an 1.1.0-M for Pekko-Persistence-JDBC. Background of my question is that we are currently in a migration phase from Akka to Pekko and we would like to jump onto the 1.1.0 'release train' if possible.

**Comment by pjfanning** [comment]
It will be a couple of months because we have to release a lot of modules that this one relies on.
I would recommend using Pekko 1.0. That is actually the release that was designed to be the one that is easiest for Akka users to migrate to.
We haven't heard of any issues with Pekko 1.1.0-M1 but there is a bug in the Pekko Core 1.1.0 release so you should avoid that (in pekko-cluster-sharding).

**Comment by janjaali** [comment]
Assuming with the latest releases we can close this discussion now. Thanks for all the contributions that you make to Pekko!
